facilities/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from common.models import AbstractBase, SubLocation
+
+FACILLITY_STATUS = (
+    ('OPERATIONAL', 'Operations are running normally'),
+    ('NOT_OPERATIONAL', 'The facility is not operating'),
+)
+
+FACILITY_TYPES = (
```

Comment:
This should be a model, with default, data.

One of the big failings of MFL 1.0 was the lack of configurability of changeable choices like this.